### PR TITLE
[Need review]b04505049 黃子昇

### PR DESCRIPTION
**Pull request by wongchiseng**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is there any suggestion?

<br />

### CODE REVIEW

**Comment by amjltc295** [comment]
Exceeds deadline.

**Comment by amjltc295** [comment]
-20 for submission failure.

File: src/students/b04505049.py [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+    from urllib import request
+    result_img = None
+
+    
+
+    # TODO: download the image from img_url with the request module
+    # and add your student ID on it with draw_name() in the utils module
+    # under src/.
+
+    request.urlretrieve(img_url, "Arianrhod.jpg")
```
Comment (amjltc295) [inline src/students/b04505049.py]: You are not allowed to use the local image.

Comment (wongchiseng) [reply src/students/b04505049.py]: ok

Comment (wongchiseng) [reply src/students/b04505049.py]: fixed

Comment (wongchiseng) [reply src/students/b04505049.py]: 剛才收到X,沒通過驗證。己重新fix。

**Pull request merged**